static/app/views/organizationContext.tsx

```diff
@@ -82,8 +90,15 @@ export function OrganizationContextProvider({children}: Props) {
       return;
     }
 
+    const logSet = () => {
```

Comment:
```suggestion
    const updateLoadingState = () => {
```
nit -- can we rename this function or just define it inline for the promise chain (this is already a nested anonymous function, so we won't get any hot process wins by naming it)

Reply:
whoops I think I did this for testing with logs and forgot to switch it back